lib/tasks/import_old_service_manual_content.rake

```diff
@@ -0,0 +1,102 @@
+require "English"
+
+def all_old_guides
+  directory = File.join(Dir.tmpdir, "government-service-design-manual", "git")
+  unless Dir.exist?(directory)
+    print "Cloning repository..."
+    sh "git clone --depth 1 https://github.com/alphagov/government-service-design-manual #{directory}"
+    puts " [done]"
+  end
+
+  objects = Dir.glob("#{directory}/service-manual/**/*.md")
+  .map do |path|
+    url = path.gsub(directory, "")
+    url = url.gsub(".md", "")
+    url = url.gsub(/\/index$/, '')
+    {
+      path: path,
+      url: url,
+    }
+  end
+
+  # https://github.com/jekyll/jekyll/blob/2807b8a012ead8b8fe7ed30f1a8ad1f6f9de7ba4/lib/jekyll/document.rb
+  yaml_front_matter_regexp = /\A(---\s*\n.*?\n?)^((---|\.\.\.)\s*$\n?)/m
+
+  objects.map do |o|
+    content = File.read(o[:path])
+    title = ""
+
+    if content =~ yaml_front_matter_regexp
+      body = $POSTMATCH
+      data_file = YAML.load($1)
+      title = data_file["title"]
+      state = data_file["status"] || "draft"
+      {
+        url: o[:url],
+        path: o[:path],
+        title: title,
+        state: state,
+        body: body,
+      }
+    end
+  end
+
+end
+
+desc "import old content from government-service-design-manual"
+task import_old_service_manual_content: :environment do
+  author = User.first || User.create!(name: "Unknown", email: "unknown@example.com")
```

Comment:
Isn't this the default user of the system? Don't we need this to be able to log into the app for the first time after a fresh db:reset?